protocol/msgpack/unaggregated_iterator.go

```diff
@@ -40,18 +40,18 @@ const (
 type unaggregatedIterator struct {
 	iteratorBase
 
-	ignoreHigherVersion bool
-	largeFloatsSize     int
-	largeFloatsPool     pool.FloatsPool
-	iteratorPool        UnaggregatedIteratorPool
+	largeFloatsSize int
+	largeFloatsPool pool.FloatsPool
+	iteratorPool    UnaggregatedIteratorPool
 
-	closed             bool
-	metric             unaggregated.MetricUnion
-	policiesList       policy.PoliciesList
-	id                 id.RawID
-	timerValues        []float64
-	cachedPolicies     [][]policy.Policy
-	cachedPoliciesList policy.PoliciesList
+	ignoreHigherVersion bool
```

Comment:
Yep, will update. I moved it next to the other bool to minimize the size of the struct (since we pool these I think it could be worthwhile). But I think I made this change before we talked about struct field ordering. Will move it to the group above.